changelog.d/20230707_183519_ollie_scriv.md

```diff
@@ -0,0 +1,3 @@
+### Added
+
+- `Rel8.materialize` and `Rel8.Tabulate.materialize`, which add a materialization/optimisation fence to `SELECT` statements by binding a query to a `WITH` subquery. Note that Rel8 doesn't currently add `MATERIALIZE` to this, but may in the future. ([#180](https://github.com/circuithub/rel8/pull/180))
```

Comment:
The text itself is fine but I thought we were going to leave this out for now?

Reply:
Oh, I thought I removed this entry and only talked about `loop`. But yes, I don't plan to mention this in the changelog.